docs/content/install.sh

```diff
@@ -152,9 +152,13 @@ case $OS in
     chown root:root /usr/bin/rclone.new
     mv /usr/bin/rclone.new /usr/bin/rclone
     #manuals
-    mkdir -p /usr/local/share/man/man1
-    cp rclone.1 /usr/local/share/man/man1/
-    mandb
+    if ! [ -x "$(command -v mandb)" ]; then
```

Comment:
This runs the mandb command twice which isn't desirable since it can take quite a long time...

How about using the detection we use above for discovering the zip tools?

```sh
    trash=`hash $tool 2>>errors`
    if [ "$?" -eq 0 ]; then
        unzip_tool="$tool"
        break
    fi
```

Reply:
Sure I can do that, but I don't think it runs the command twice, I think it only checks that it exists.
if you replace mandb with a script that has a side effect such as deleting a file, you'll see that it doesn't run

Reply:
Sorry I mis-read the man page you are right.

I'll merge what you've done now :-)